conceptpower/classes.py

```diff
@@ -157,4 +158,60 @@ def get_type(self, uri):
                 data['supertype_id'] = snode.get('supertype_id')
                 data['supertype_uri'] = snode.get('supertype_uri')
         return data
-        
\ No newline at end of file
+
+    def create(self,user,password,required_concept_data,**kwargs):
+        """
+        Adds a concept to the conceptlist by using rest api
+
+        Parameters
+        -----------
+        user: str
+             userId of the person who wants to add concept
+        password: str
+                 password of the person who wants to add concept
+        required_values: dict
+                        required keys
+                        key: "word"
+                                name of the word
+                        key: "pos"
+                                part of speech of the word
+                        key: "conceptlist"
```

Comment:
I wouldn't worry too much about the number of arguments; the signature can span multiple lines, so it will still look neat.  But that is a good point about the payload for the request.

We should either explicitly state the arguments in the signature, or just use the ``**kwargs`` collector.